tests/test_request_timeout.py

```diff
@@ -80,18 +80,32 @@ def __init__(self, *args, **kwargs):
         self._post_connect_delay = _post_connect_delay
         self._pre_request_delay = _pre_request_delay
 
-    @asyncio.coroutine
-    def connect(self, req):
-        d_req = DelayableTCPConnector.\
-            RequestContextManager(req, self._pre_request_delay)
-        conn = yield from super(DelayableTCPConnector, self).connect(req)
-        if self._post_connect_delay and self._post_connect_delay > 0:
-            _ = yield from asyncio.sleep(self._post_connect_delay,
-                                         loop=self._loop)
-        req.send = d_req.send
-        t = req.loop.time()
-        print("Connected at {}".format(t), flush=True)
-        return conn
+    if aiohttp.__version__ >= '3.0':
+        @asyncio.coroutine
```

Comment:
why are you using this syntax vs `async def`?

Reply:
@r0fls it was there before, i just moved it into the `if`, but we can change it to `async def`

Reply:
@r0fls done, interesting travici is failing, but it's passing here https://travis-ci.org/yunstanford/sanic

Reply:
Ya weird, I thought this was passing before, and that change shouldn't impact it... let me try rerunning it

Reply:
yeah, it's passing both locally and here https://travis-ci.org/yunstanford/sanic..